sdk/eventhub/azure-eventhub/azure/eventhub/_eventprocessor/event_processor.py

```diff
@@ -48,12 +49,11 @@ class EventProcessor(
 
     def __init__(
         self,
-        eventhub_client,  # type: EventHubConsumerClient
-        consumer_group,  # type: str
-        on_event,  # type: Callable[[PartitionContext, Union[Optional[EventData], List[EventData]]], None]
-        **kwargs  # type: Any
-    ):
-        # type: (...) -> None
+        eventhub_client: EventHubConsumerClient,
+        consumer_group: str,  
```

Comment:
Looks like there are trailing whitespaces in many places. You'll need to remove these to pass CI. Ex.
```suggestion
        consumer_group: str,
```
[[Here](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1849973&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=56d4e2e6-c43b-527c-bad7-234ebe7429dd&l=461)] is the list of all the places that have trailing whitespace.